chaos_monkey/monkey_utils/sambacry_monkey_runner/sc_monkey_runner.c

```diff
@@ -0,0 +1,168 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+#include "sc_monkey_runner.h"
+
+#ifdef __x86_64__
+	#define ARC_IS_64
+#endif
+
+#ifdef _____LP64_____
+	#define ARC_IS_64
+#endif
+
+#define LINE_MAX_LENGTH (2048)
+#define MAX_PARAMETERS (30)
+
+int samba_init_module(void)
+{
+#ifdef ARC_IS_64
+	const char RUNNER_FILENAME[] = "sc_monkey_runner64.so";
+	const char MONKEY_NAME[] = "monkey64";
+	const char MONKEY_COPY_NAME[] = "monkey64_2";
+#else
+	const char RUNNER_FILENAME[] = "sc_monkey_runner32.so";
+	const char MONKEY_NAME[] = "monkey32";
+	const char MONKEY_COPY_NAME[] = "monkey32_2";
+#endif
+	const char RUNNER_RESULT_FILENAME[] = "monkey_runner_result";
+	const char COMMANDLINE_FILENAME[] = "monkey_commandline.txt";
+	const char ACCESS_MODE_STRING[] = "0777";
+	const char RUN_MONKEY_CMD[] = "sudo ./";
+	
+	int found = 0;
+	char modulePathLine[LINE_MAX_LENGTH];
+	char commandline[LINE_MAX_LENGTH] = {'\0'};
+	char* monkeyDirectory;
```

Comment:
You don't initialize all variables, without being consistent on why.

Reply:
fixed